pre_commit/commands/run.py

```diff
@@ -371,6 +371,11 @@ def run(
         environ['PRE_COMMIT_FROM_REF'] = args.from_ref
         environ['PRE_COMMIT_TO_REF'] = args.to_ref
 
+    # Exposes remote branch ref as an environment variable
+    # for pre-push hooks consumption
+    if args.remote_branch:
+        environ['PRE_COMMIT_REMOTE_REF'] = args.remote_branch
```

Comment:
this should be combined with the block below -- the naming here is inconsistent as well

Reply:
I updated the code as recommended.